### ISSUE TITLE
[DOCS] Monitoring with metricbeat page has incorrect setting

### ISSUE THREAD

**Issue by robbavey** [elastic/logstash#11596]
[Documentation](https://www.elastic.co/guide/en/logstash/current/monitoring-with-metricbeat.html) on how to monitor Logstash with metricbeat, references an invalid setting when discussing how to [disable the default collection of monitoring metrics](https://www.elastic.co/guide/en/logstash/current/monitoring-with-metricbeat.html#disable-default):

The documentation states to set `monitoring.enabled` to `false`. This is not currently a valid setting in `logstash.yml` and should read `xpack.monitoring.enabled` instead.

Attempting to set `monitoring.enabled: false` will result in the following error:

```
[2020-02-11T14:22:51,678][FATAL][logstash.runner          ] An unexpected error occurred! {:error=>#<ArgumentError: Setting "monitoring.enabled" hasn't been registered>
```

This issue affects documentation versions from `7.3` onwards.

cc @karenzone

**Comment by karenzone** [comment]
Fixed with #11597

**Comment by naj-h** [comment]
@robbavey @karenzone Surprisingly our docs still mention `monitoring.enabled: false`. It actually did not trigger the above error `An unexpected error occurred!`, but if it's not the correct setting then we need to fix it.

https://www.elastic.co/guide/en/logstash/current/monitoring-with-metricbeat.html
